👌 IMPROVE: Teams

ahmadawais opened the pull request:
Updated the teams on my profile. I have both provided edits to the content and just contributed to as a developer to the repo.

Peace! ✌️

rviscomi commented:
Thanks for your contributions @ahmadawais. To clarify, there's a fair amount of work involved on the Editors team. See this [copyediting](https://github.com/HTTPArchive/almanac.httparchive.org/issues/218) issue for example. If you're interested in joining I have a few chapters I can give you 😁

ahmadawais commented:
@rviscomi thanks for sharing that with me. I was definitely not aware of all the work these awesome editors have provided us with. 

1. I have removed the `editors` team so you can merge this
1. I'd love to copyedit the `Media` chapter
1. Removed myself from the `editors` team because I just landed here after 36 hours of travel, while I'd love to copyedit, there's a chance I might not be able to find time this close to the deadline. So, yeah — going to help either way.

Peace! ✌️
